### DISCUSSION TITLE
Python version update

### DISCUSSION THREAD

**Discussion by AndreaPra** [stellargraph/stellargraph#2093] [General]
Hi,

it's the first time I use this library, I am currently working on my master's thesis which is about graph ML. While looking for a nice library I found this one, which seems promising. Unfortunately, I have to use a **python version which is > 3.9.0**, therefore, even if I would like to use this library I cannot. 

So, the question is, do you plan to update the python version requirement?

Thank you :)

**Comment by mvisani** [comment]
Hey ! 

I had the same problem as you ! I ended up forking their repository and changed the requirements myself. If you don't want to bother installing from source I created my own package that installs the library with the latest python version. Tell me if it doesn't work ;) 

`pip install stellargraph-mvisani`
